docs/howto/initialize-database.rst

```diff
@@ -23,9 +23,8 @@ Initializing a Database Backend
 If you want to take a real test drive of Airflow, you should consider
 setting up a real database backend and switching to the LocalExecutor.
 
-As Airflow was built to interact with its metadata using the great SqlAlchemy
-library, you should be able to use any database backend supported as a
-SqlAlchemy backend. We recommend using **MySQL** or **Postgres**.
+Airflow was built to interact with its metadata using SqlAlchemy
+with **MySQL** and **Postgres** as supported backends.
```

Comment:
```suggestion
with **MySQL**,  **Postgres** as **SQLite** supported backends.
```

Reply:
maybe not making perfect sense to me, correct me if I'm wrong @potiuk :

This page is meant for users who have already installed Airflow and tried with SQLite (the earlier paragraph is `If you want to take a real test drive of Airflow, you should consider setting up a real database backend and switching to the LocalExecutor.`)

So I would like to suggest to apply your change + append with "`(SQLite is used primarily for development purpose)`". In addition I think you missed the "`as`" in your change ;-)

Sorry for being quite picky for documentation. It's quite essential to me though.

Reply:
Well. There are anecdotal evidences of people using SQLite and sequential executor in production :). 
And you know @XD-DENG, And if you look closely, SQLite is one of the very few "True Serverless" databases https://sqlite.org/serverless.html , so in the  "Serverless" hype maybe everyone should switch to it ;) 

But on a more serious note - yeah. I think adding a note is a good idea  :)

Reply:
I do like SQLIte as well.

The sentence I suggested to add is actually copied from current version of README. So it should be consistent